Tella/Scenes/Settings/Views/Servers/GDrive/CreateDriveFolder.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CreateDriveFolder.swift
+//  Tella
+//
+//  Created by gus valbuena on 5/21/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct CreateDriveFolder: View {
+    @State var fieldContent : String = ""
+    @State var isValid : Bool = false
+    var body: some View {
+        ContainerView {
+            VStack(spacing: 20) {
+                Spacer()
+                ServerConnectionHeaderView(
+                    title: "Create new folder",
+                    subtitle: "Your reports will be uploaded to a new folder on your Google Drive. Choose a name for this folder here."
+                )
+                TextfieldView(fieldContent: $fieldContent,
+                              isValid: $isValid,
+                              shouldShowError: .constant(false),
+                              fieldType: .text,
+                              placeholder: "Folder name")
+                    .padding(.vertical, 12)
+                Spacer()
+                bottomView
+            }.padding(EdgeInsets(top: 0, leading: 15, bottom: 0, trailing: 15))
+            .navigationBarHidden(true)
+        }
+    }
+    
+    var bottomView: some View {
+        BottomLockView<AnyView>(isValid: $isValid,
+                                nextButtonAction: .action,
+                                shouldHideNext: false,
+                                shouldHideBack: false,
+                                nextAction: {
+            navigateTo(destination: SuccessLoginView(navigateToAction: {}, type: .gDrive))
+        })
+
+    }
+}
+
+#Preview {
+    CreateDriveFolder()
+}
```

Comment:
Do we really need the previews (also in the other views)!? the canvas is not working in any preview ❌